base_layer/core/src/test_helpers/blockchain.rs

```diff
@@ -45,10 +73,199 @@ pub fn create_new_blockchain() -> BlockchainDatabase<MemoryDatabase<HashDigest>>
         })
         .on_ties(ChainStrengthComparerBuilder::new().by_height().build())
         .build();
-    super::create_mem_db(&consensus_manager)
+    let validators = Validators::new(MockValidator::new(true), MockValidator::new(true));
+    create_store_with_consensus_and_validators(&consensus_manager, validators)
 }
 
 fn genesis_template() -> NewBlockTemplate {
     let header = BlockHeader::new(0);
     header.into_builder().build().into()
 }
+
```

Comment:
Suggestion - A Builder pattern would be easier to extend with additional features and variations.